src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -344,7 +344,8 @@ class queue_cb {
         /* to enable aio batching, the stream needs to have its own context
          * and any of the 
          */
-        if (qAioEn && (byteThresh || pktThresh)) {
+        if (qAioEn && (byteThresh || pktThresh) && !qAioBatchEn) {
+            std::lock_guard<std::mutex> lk(reqLock);
```

Comment:
yeah, oh well, this is a work-around for a false positive Coverity defect, since this function is called from main thread only before worker thread shares qExit.